cgroups/fs/apply_raw.go

```diff
@@ -161,6 +161,20 @@ func (m *Manager) Set(container *configs.Config) error {
 	return nil
 }
 
+func (m *Manager) AddProcess(pid int) error {
+	for name, path := range m.Paths {
+		_, ok := subsystems[name]
```

Comment:
Yeah, you're right it is different logic. Maybe wrapper with proper name around `writeFile(path...` will help me in future to distinct such cases.